test/e2e/generate_kube_test.go

```diff
@@ -849,6 +849,62 @@ var _ = Describe("Podman kube generate", func() {
 		Expect(inspect.OutputToString()).To(ContainSubstring(vol1))
 	})
 
+	It("with subpath volume", func() {
+		var args []string
+		volName := "vol-test1"
+		mkvol := podmanTest.Podman([]string{"volume", "create", volName})
+		mkvol.WaitWithDefaultTimeout()
+		Expect(mkvol).Should(ExitCleanly())
+
+		mountPoint := "/mnt"
+		filepath1 := "etc/etcfile"
+		dir1 := filepath.Join(mountPoint, "etc")
+		inFile1 := filepath.Join(mountPoint, filepath1)
+		exec1 := fmt.Sprintf("mkdir -p %s; touch %s", dir1, inFile1)
+
+		filepath2 := "var/varfile"
+		dir2 := filepath.Join(mountPoint, "var")
+		inFile2 := filepath.Join(dir2, "varfile")
+		exec2 := fmt.Sprintf("mkdir -p %s; touch %s", dir2, inFile2)
+
+		popvol := podmanTest.Podman([]string{"run", "-v", volName + ":" + mountPoint,
```

Comment:
you can now use `podmanTest.PodmanExitCleanly()` for calls like this. See https://github.com/containers/podman/blob/main/test/e2e/pull_chunked_test.go#L77

Reply:
That indeed reduced quite a bit of checks. Thanks!